app/views/articles/articles/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<p style="color: green"><%= notice %></p>
+
+<h1>Articles</h1>
+
+<div id="Aeticles">
+  <% @articles.each do |article| %>
+    <%= render article %>
+    <p>
+      <%= link_to "Show this article", article %>
+    </p>
+  <% end %>
```

Comment:
<%= render @articles %> - Loop is not required here.

Pls refer the topic below:
**Rails render collection**

Reply:
we need relative show button....So I cant use render collection. We are using same partial for show page also cant move show button